metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.vysp3r.ProtonPlus</id>
+    <metadata_license>FSFAP</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <content_rating type="oars-1.1" />
+
+    <name>ProtonPlus</name>
+    <developer_name>Vysp3r</developer_name>
+    <summary> A simple Wine and Proton-based compatiblity tools manager for GNOME </summary>
+    <description>
+        <p>ProtonPlus a simple Wine and Proton-based compatiblity tools manager for GNOME. It works with Steam, Lutris, Heroic Games Launcher and Bottles. It was made in Vala with GTK4 and Libadwaita using GNOME Builder.</p>
```

Comment:
since this is "Wine" shouldn't the build be x86_64 only?

Reply:
indeed, I fixed it

Reply:
doesn't look like it.

Reply:
The appdata file is generated when the application is built with my current config. What do I need to do to make it work then?

Reply:
1. fix it upstream.
2. apply the patch until there is a new release.

Reply:
What do you mean by fixing it upstream?

Reply:
> The appdata file is generated when the application is built with my current config.

thats upstream.

Reply:
So I fix it in my project repo and create a new release with a working appdata file?

Reply:
@hfiguiere The file was fixed upstream and I updated the manifest to make sure it's using the latest version including the fix.

Reply:
> since this is "Wine" shouldn't the build be x86_64 only?

I just realised that yes for now it's mainly for Proton and Wine, but if some tools get out for arch64 in the future I will add them for sure so excluding the arch64 build is not necessary.